diff --git a/pyproject.toml b/pyproject.toml
index 6dae31d..558d8fe 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -8,7 +8,7 @@ dynamic = ["version"]
 
 [dependency-groups]
 dev = [
-    "omni-core-rs",
+    # "omni-core-rs",
     "omni-dev-fusion-agent",
     "omni-dev-fusion-common",
     "polyfactory>=3.2.0",
@@ -30,7 +30,7 @@ source = "vcs"
 [tool.uv.sources]
 omni-dev-fusion-common = { workspace = true }
 omni-dev-fusion-agent = { workspace = true }
-omni-core-rs = { path = "packages/rust/bindings/python", editable = true }
+# omni-core-rs = { path = "packages/rust/bindings/python", editable = true }
 
 
 [tool.uv.workspace]
